bin/check.sh

```diff
@@ -11,7 +13,9 @@ gometalinter --concurrency=4 --enable-gc --deadline=300s --disable-all\
   --enable=goconst\
   --enable=gofmt\
   --enable=goimports\
-  --enable=golint --exclude=.pb.go\
+  --enable=golint\
+  --exclude=.pb.go\
+  --exclude=_test.go\
```

Comment:
I was trying to avoid adding special cases for specific auto-generating mock test files, but maybe it's not a big deal (yet)? I can add a specific exclusion for mock_config_test.go and we can generalize later based on some naming scheme, e.g. "--exclude=_autogen_test.go". Thoughts?